listenbrainz/webserver/static/js/src/RecentListens.tsx

```diff
@@ -83,6 +90,8 @@ export default class RecentListens extends React.Component<
       previousListenTs: props.listens?.[0]?.listened_at,
       direction: "down",
       recordingFeedbackMap: {},
+      offset: 0,
```

Comment:
`page` based pagination would be easier than `offset` I think. We use it on the Reports page, so it would be good for consistency too.